### ISSUE TITLE
Error in modelsearch exhaustive with iiv_strategy=3

### ISSUE THREAD

**Issue by schaedeli** [pharmpy/pharmr#68]
Hi Rikard,
I ran into another error during the modelsearch tool testing, when trying the algorithm "exhaustive":

```
> sp1 <- "TRANSITS([1,10],*);PERIPHERALS(1);ABSORPTION(SEQ-ZO-FO)"
> # 1-exhaustive - modelsearch_dir 
> res_modelsearch_1 <- run_tool("modelsearch", 
+                             model=input_model,
+                             search_space=sp1, 
+                             algorithm="exhaustive", # "reduced_stepwise", 
+                             rankfunc="bic", 
+                             iiv_strategy=3)
C:\PROGRA~1\Python39\lib\site-packages\distributed\node.py:160: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 55005 instead
  warnings.warn(
distributed.worker - WARNING - Compute Failed
Function:  _add_iiv_to_func
args:      ('ABSORPTION(SEQ-ZO-FO)', 3.0, <Pharmpy model object modelsearch_candidate17>)
kwargs:    {}
Exception: TypeError('_add_iiv_to_func() takes 2 positional arguments but 3 were given')

```
This error appears with the search algorithm “exhaustive” and iiv_strategy=3, independent of the search_space, i.e. also when the search space is reduced to TRANSITS and PERIPHERALS. The run fails. Is this a known bug / issue ?

Kind regards
Franziska

**Comment by stellabelin** [comment]
Hi Franziska, thanks for reporting! It was not a known bug/issue, but I can recreate the issue on another dataset so I can confirm this is a bug.

**Comment by stellabelin** [comment]
Hi! This was fixed in pharmpy/pharmpy@52a99b6b0715d2735c836d73598081e81bc2c432 (and forgot to close this issue), closing this issue!
